main-app/app/scripts/authentication/authenticationService.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.BingoClient.Authentication')
+        .service('tokenService', function () {
```

Comment:
Close, but extend this out to cover the api calls - this will simplify your code and encapsulate into one place.